satellite/referrals/service.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package referrals
+
+import (
+	"context"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/rpc"
+	"storj.io/storj/pkg/signing"
+	"storj.io/storj/pkg/storj"
+)
+
+var (
+	// Error is the default error class for referrals package.
+	Error = errs.Class("referrals")
```

Comment:
we already have named logger ```referrals:service```. In this case main error class name is not very meaningful